version-control/git/collaborate/Readme.md

```diff
@@ -43,14 +46,15 @@ discuss potential modifications, and even push follow-up commits if necessary.
 ### Exercise : Issue a Pull Request and Review it
 
 In this exercise, "Beta" will be making changes and submitting a pull request
-to Alpha's repository, and "Alpha" will be reviewing the pull request and merging
-it into their own remote (and then local) repository.  In the following instructions
-Beta will be performing Steps 1-4 (while Alpha waits and observes), and then
-Alpha will continue with Steps 5-7 (while Beta observes).  
+to the upstream repository, and "Alpha" will be reviewing the pull request and
```

Comment:
We will need to be sure all of the students have write access to the McMahon Lab repo in question(adding them to the org and a team for this repo)